dbscripts/xml/install.xml

```diff
@@ -18,6 +18,7 @@
 	<!-- PKP schema components -->
 	<migration class="PKP\migration\install\CommonMigration" />
 	<migration class="PKP\migration\install\ControlledVocabMigration" />
+	<migration class="PKP\migration\install\FailedJobsMigration" />
```

Comment:
This will take care of installation, but the table will also need to be added during upgrade. (Also OMP and OPS)